s2apler/model.py

```diff
@@ -512,7 +512,6 @@ def predict(
                 else:
                     block_dict_no_null_titles[block_key].append(paper_id)
         block_dict = dict(block_dict_no_null_titles)
-        block_dict_no_null_titles = dict(block_dict_null_titles)
```

Comment:
Removing this swapped variable naming to prevent confusion

Reply:
It also appears in rows 513 and 514. Won't those rows error out?

Reply:
Nevermind, I see what's happening. Thanks.